syntax-config.pri

```diff
@@ -7,11 +7,16 @@ win32|macx {
         DEFINES += GTKSOURCEVIEW_LANGUAGE_PATH=\\\"/usr/share/gtksourceview-3.0/language-specs/\\\"
         message(Using GtkSourceView language specifications.)
     }
+    
+    DATA_INSTALL_PREFIX = $$LIRI_INSTALL_DATADIR/liri-text
+    DATA_RUNTIME_PREFIX = $$CUSTOM_RUNTIME_DATADIR/liri-text
+    isEmpty(DATA_RUNTIME_PREFIX) {
```

Comment:
Why not to simply pass DATA_RUNTIME_PREFIX in special cases like snap packaging?